packages/@pollyjs/ember/index.js

```diff
@@ -2,18 +2,49 @@
 
 'use strict';
 
+const fs = require('fs');
 const path = require('path');
 const { registerExpressAPI, Defaults } = require('@pollyjs/node-server');
+const parseArgs = require('minimist');
 
-const { assign } = Object;
+function determineEnv() {
```

Comment:
Lifted from a previous method I wrote for ember-cli-dotenv:
https://github.com/fivetanley/ember-cli-dotenv/blob/master/index.js#L49-L68